test/correctness/func_clone.cpp

```diff
@@ -159,14 +159,6 @@ int update_defined_after_clone_test() {
         return 1;
     }
 
-    Buffer<int> im = g.realize({200, 200});
```

Comment:
Prefix this block of code with param.set(false) instead of deleting it.

Reply:
Done